src/containers/App.tsx

```diff
@@ -66,12 +66,12 @@ function App() {
           />
           <Route
             exact
-            path="/patients/edit/:ccpId/:patientId"
+            path="/events/:eventId/ccps/:ccpId/patients/edit/:patientId"
```

Comment:
this is the pattern we're gonna stick to
```suggestion
            path="/events/:eventId/ccps/:ccpId/patients/:patientId/edit"
```